po/be.po

```diff
@@ -302,11 +302,11 @@ msgstr "Калі ласка абнавіцеся да апошняй версі
 msgid "Unknown exception occurred"
 msgstr "Адбылася невядомая памылка"
 
-#: data/ui/AboutDialog.ui.in:13
-msgid "Visit Piper’s website"
+#: data/ui/AboutDialog.ui.in:12
+msgid "Visit Piper on GitHub"
```

Comment:
ok, I freely admit I'm not too well-versed with translations to know this but: isn't this superfluous? and possibly covering up that we need a translation here? Those files are supposed to be re-generated by `ninja` (see the `po/README.md` file) and by updating this bit here, you're effectively hiding that there were changes. Without this commit, the next translation build will show the changed messages in the git diff, making it easier to identify.

But then again, I might be completely wrong about this here :)

Reply:
Oh I haven't checked the README actually, didn't think to much about the translations. Thanks for the heads up, will look into it :+1:

Reply:
Ok so if I get it right, we don't update any translations files, because that's done automatically when running `piper-update-po`, meaning I can simply remove the last commit. On the other hand, are the translations working without the changes?
Is there a way to "force" a specific language? Don't want to change my system language just to test this.
Same problem will occur in #448 btw.

Reply:
```
$ export LANG=de_DE.UTF-8
$ piper
```
Doesn't appear work with `piper.devel` run from in-tree unfortunately, the lookup path is wrong. And I can't find whatever meson magic is required to make it work. So you do need to install and run from the prefix dir.

Reply:
Doesn't work for me, stays english :/
But since #448 will require new translations anyway I will just remove the last commit. Is there any release schedule for Piper? Since this commit does nothing super important we could just merge it with #448, so we don't break any translations in a new release.

Reply:
@Hjdskes sorry for pinging you, do you remember how to deal with this?

Reply:
With "this", do you mean how to run with in-tree translations?

The lookup path is set to `localedir`, which is [set to `po` for devel builds](https://github.com/libratbag/piper/blob/master/meson.build#L105). I vaguely remember this working for me when I first added translations to Piper. You could see what `localedir` points to in `piper.devel`; it should point to the local `po` folder.

Reply:
Yes, thanks!

Reply:
> Doesn't work for me, stays english :/

Ahhh I don't have any languages installed, that's why it didn't work. Whatever, doesn't matter anyway.